tests/onnxruntime/test_modeling.py

```diff
@@ -2083,7 +2084,7 @@ def test_compare_to_transformers(self, test_name: str, model_arch: str, use_cach
         set_seed(SEED)
         transformers_model = AutoModelForCausalLM.from_pretrained(model_id)
         transformers_model = transformers_model.eval()
-        tokenizer = get_preprocessor(model_id)
+        tokenizer = AutoTokenizer.from_pretrained(model_id, use_fast=False if model_arch == "mistral" else True)
```

Comment:
For the tests we use tiny models, here https://huggingface.co/hf-internal-testing/tiny-random-MistralForCausalLM which cannot be loaded without adding `use_fast=False`, opened a PR to fix it https://huggingface.co/hf-internal-testing/tiny-random-MistralForCausalLM/discussions/6 (cc @ydshieh), but will replace it with https://huggingface.co/echarlaix/tiny-random-mistral so that we can merge